bin/async_capture.py

```diff
@@ -116,7 +116,7 @@ def process_capture_queue(self) -> None:
 
                 if send_report:
                     self.lookyloo.send_mail(uuid, email=settings.get('email', ''),
-                                            comment=settings.get('comment'))
+                                            comment=settings.get('comment'), email_prio= settings["email"] if settings["email"] else None)
```

Comment:
if `email` isn't a key in the dict, it will raise an exception. You should use `get` the same way we do for the `comment` (get returns none if it doesn't exists)